src/algorithms/MathUtil.java

```diff
@@ -59,11 +97,16 @@ public static int abs(int n) {
      * @param number the parameter to be revered
      * @return reversed number
      */
-    public static int reverse(int number) {
-
-        return 0;
+    public static int reverse(int number) {//I have not be able to solve this problem using recursion.
```

Comment:
static int result = 0;

public static int reverse(int number) {
    if (number <= 0) {
        return -1;
    }

    int lastDigit = number % 10;

    result = result * 10 + lastDigit;
    reverse(number / 10);
    return result;
}

Try to understand the logic of this method and try to rewrite